ddtrace/internal/datastreams/__init__.py

```diff
@@ -1,14 +1,23 @@
-from . import kafka  # noqa:F401
+from ddtrace import config
+from ddtrace import tracer
 
+from ...internal.utils.importlib import require_modules
 
+
+required_modules = ["confluent_kafka"]
```

Comment:
I copied this pattern from contrib, is it necessary/correct?

Reply:
Cool! It makes sense here as far as I can tell.